app/code/Magento/Checkout/etc/adminhtml/system.xml

```diff
@@ -41,6 +41,10 @@
                 <field id="number_items_to_display_pager" translate="label" type="text" sortOrder="2" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
                     <label>Number of Items to Display Pager</label>
                 </field>
+                <field id="crosssell_enabled" translate="label" type="select" sortOrder="3" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                    <label>Show Crosssell Items in the Shopping Cart</label>
```

Comment:
Please add the translation to CSV for the new phrase.

Reply:
@VladimirZaets  Thank you for your feedback, I have added the translation.